week3/homework/src/index.js

```diff
@@ -1,3 +1,74 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const {
+  readAndParse,
+  create,
+  update,
+  deleteToDo,
+  showOneToDo,
+  mark,
+  handleError,
+} = require('./actions');
+
+const express = require('express');
+const app = express();
+
+const PORT = 3000;
+
+app.use(express.json());
+
+app.get('/', (request, response) => {
+  readAndParse('help')
+    .then(help => response.json(help))
+    .catch(error => response.status(error.statusCode || 500).json(handleError(error)));
```

Comment:
This part seems to be repeating in every call. There is a way to eliminate the duplication:
https://expressjs.com/en/guide/error-handling.html
This is not part of the homework but might be interesting for you to know.